src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/MessageOperationsTest.java

```diff
@@ -50,21 +51,22 @@ void complete_should_throw_exception_when_queue_client_fails() throws Exception
     }
 
     @Test
+    @Disabled
```

Comment:
These tests are removed in https://github.com/hmcts/bulk-scan-orchestrator/pull/385

Reply:
That PR is merged now. Can we remove or update the tests accordingly ?

Reply:
I have to update EnvelopeEventProcessor tests as well with the latest changes. Will update the pr soon